documentation/Using-the-SDK/Creating-your-plugin.md

```diff
@@ -222,8 +223,87 @@ public class MyProcessingSource : ProcessingSource
 
 Currently, `MyDataProcessor` does not exist and we do not know which arguments to pass in. This is because the implementation of `MyDataProcessor` depends on the plugin framework you choose in the [Choosing a Plugin Framework](#choosing-a-plugin-framework) step. We will revisit this method once we have created `MyDataProcessor`.
 
-At this point, our `ProcessingSource` is almost complete. Before we continue, however, we must decide on which plugin framework our plugin will use.
+At this point, our `ProcessingSource` is almost complete. Before we continue, however, we must [choose a plugin framework](#choosing-a-plugin-framework).
 
+### Adding About Information
+
+When a plugin is used by a large and/or public audience, it is useful to 
+give users an easy way to contact its owners/maintainers. SDK Drivers such as 
+WPA look for __about information__ on a `ProcessingSource` for it to  
+present to users. Note adding about information is optional.
+
+To add about information to your `ProcessingSource`, simply override or 
+implement `GetAboutInfo`:
+
+```cs
+[ProcessingSource(...)]
+public class MyProcessingSource
+    : ProcessingSource
+{
+    // ...
+
+    public override ProcessingSourceInfo GetAboutInfo()
+    {
+        return new ProcessingSourceInfo
+        {
+            Owners = new[]
+            {
+                new ContactInfo
+                {
+                    Name = "Author Name",
+                    Address = "Author Email",
+                    EmailAddresses = new[]
+                    {
+                        "owners@mycompany.com",
+                    },
+                },
+            },
+            LicenseInfo = null,
+            ProjectInfo = null,
+            CopyrightNotice = $"Copyright (C) {DateTime.Now.Year}",
+            AdditionalInformation = null,
+        };
+    }
+}
+```
+
+or
+
+```cs
+[ProcessingSource(...)]
+public class MyOtherProcessingSource
```

Comment:
I vote we just get rid of this since
1) It's the first time we're referencing the `IProcessingSource` interface
2) I don't even think we want most people implementing `IProcessingSource` directly.